app/assets/javascripts/utilities/buildArticleHTML.js

```diff
@@ -25,8 +25,45 @@ function buildArticleHTML(article) {
 
   if (article) {
     var container = document.getElementById('index-container');
+
+    var flareTag = '';
+    var currentTag = '';
+    if (container) {
+      currentTag = JSON.parse(container.dataset.params).tag;
+    }
+    if (article.flare_tag && currentTag !== article.flare_tag.name) {
+      flareTag =
+        "<a href='/t/" +
+        article.flare_tag.name +
+        "' class='crayons-tag'>" +
+        "<span class='crayons-story__flare-tag' style='background:" +
+        article.flare_tag.bg_color_hex +
+        ';color:' +
+        article.flare_tag.text_color_hex +
+        "'><span className='crayons-tag__prefix'>#</span>" +
+        article.flare_tag.name +
+        '</span>' +
+        '</a>';
```

Comment:
We should drop this extra container `crayons-story__flare-tag` - it's not needed anymore.

```suggestion
    flareTag = `<a href="/t/${article.flare_tag.name}" class="crayons-tag" style="background: ${article.flare_tag.bg_color_hex}; color: ${article.flare_tag.text_color_hex};">
      <span class="crayons-tag__prefix>#</span>
      ${article.flare_tag.name}
    </a>`;
```

Reply:
The flare tag is basically now a classic `crayons-tag` with custom background and color.